packages/core/cli.ts

```diff
@@ -181,6 +181,19 @@ yargs(process.argv.slice(2))
 					type: "string",
 					group: "Fuzzer:",
 					default: ["json", "text", "lcov", "clover"],
+				})
+				.option("timeout", {
+					describe: "Timeout in milliseconds for each fuzz test execution.",
+					type: "number",
+					group: "Fuzzer:",
+					default: 5000,
+				})
+				.option("max_total_time", {
```

Comment:
Shouldn't this be passed along to the fuzzer?

Reply:
I forgot to delete this--it's a remnant of an experiment. Deleted it now.